dev/TabView/TabView.cpp

```diff
@@ -231,16 +231,32 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
     if (auto listView = m_listView.get())
     {
         // Now that ListView exists, we can start using its Items collection.
-        auto items = TabItems();
-        auto numItemsToCopy = static_cast<int>(items.Size());
-        if (auto lvItems = listView.Items())
+        
+        if (!listView.ItemsSource())
```

Comment:
I am concerned about this not happening if there is an ItemsSource. In that case, there shouldn't be an TabViewItems to copy over, but we would still want TabItems to be set to lvItems so that the app has access to those items.

Reply:
That's why the else block was necessary, right? I was confused by your earlier review. 😃

Reply:
Sorry if I wasn't clear! I think an else block is necessary, but I think it should do something different. :) The TabView.TabItemsSource is bound directly to ListView.ItemsSource, so the owner of TabView already has access to those items (and was the one who provided them) -- TabView.TabItems should be a list of the realized TabViewItems, not the data items. So in the case where there's an ItemsSource, probably we just need to set TabView.TabItems = ListView.Items so that the owner can access the realized TabViewItems that ListView generates. Does that make sense?

Reply:
Understood.
So the `TabItems(lvItems)` part should be executed in both case. Then, how about this code: 
```
if (auto const lvItems = listView.Items())
{
	if (!listView.ItemsSource())
	{
		winrt::IVector<winrt::IInspectable> const itemList{ winrt::single_threaded_vector<winrt::IInspectable>() };

		for (auto const item : TabItems())
		{
			itemList.Append(item);
		}

		lvItems.Clear();

		for (auto const item : itemList)
		{
			if (item)
			{
				lvItems.Append(item);
			}
		}
	}

	TabItems(lvItems);
}
```

Reply:
I like it!!